client/src/components/AddChild/addchild.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import DeleteBtn from "../DeleteBtn";
+import Jumbotron from "../Jumbotron";
+import API from "../../utils/API";
+import { Link } from "react-router-dom";
+import { Col, Row, Container } from "../Grid";
+import { List, ListItem } from "../List";
+import { Input, TextArea, FormBtn } from "../Form";
+
+class AddChild extends Component {
+  state = {
+    child_name: "",
+    piggy: 0,
+    status: "",
+  };
+
+  handleInputChange = event => {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value
+    });
+  };
+
+  handleFormSubmit = event => {
+    event.preventDefault();
+    if (this.state.child_name && this.state.piggy) {
+      API.getAddChild({
+        child_name: this.state.child_name,
+        piggy: this.state.piggy
+      })
+      // FIXME: see .then below
+        .then(res => this.getAddChild())
+        .catch(err => console.log(err));
+    }
+  };
+
+  render() {
+    return (
+      <Container fluid>
+        <Row>
+          <Col size="md-6">
+            <Jumbotron>
+              <h1>Add child_name</h1>
+            </Jumbotron>
+            <form>
```

Comment:
Can extract out this form to it's own component and send down the functions to the props.